backend/routes/events.py

```
@@ -0,0 +1,63 @@
+        db.query(Event.id)
+        .filter(Event.event_datetime > in_30_days, Event.event_datetime <= in_1_year)
+        .all()
+    )
+
+    return {
+        "within_7_days": [e.id for e in events_within_7_days],
+        "within_30_days": [e.id for e in events_within_30_days],
+        "within_year": [e.id for e in events_within_year],
+    }
+
+@router.get("/events/by-ids/paginated", response_model=List[EventResponse])
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)